docs/manpages/pdnsutil.1.rst

```diff
@@ -47,11 +47,13 @@ algorithms are supported:
 
 activate-zone-key *ZONE* *KEY-ID*
     Activate a key with id *KEY-ID* within a zone called *ZONE*.
-add-zone-key *ZONE* {**KSK**,\ **ZSK**} [**active**,\ **inactive**] *KEYBITS* *ALGORITHM*
+add-zone-key *ZONE* {**KSK**,\ **ZSK**} [**active**,\ **inactive**] [**hidden**] *KEYBITS* *ALGORITHM*
     Create a new key for zone *ZONE*, and make it a KSK or a ZSK, with
     the specified algorithm. The key is inactive by default, set it to
     **active** to immediately use it to sign *ZONE*. Prints the id of
-    the added key.
+    the added key. The key is made visible in the zone by default, set
+    it to **hidden** to keep it from appearing, usefull for algorithm
```

Comment:
s/usefull/useful/

Perhaps detail that "visible" means "DNSKEY published in the zone" and "active" means "used to sign RRSets"?

Reply:
Fixed the typo and reworked the text a little.